lib/global-styles.php

```diff
@@ -975,5 +975,6 @@ function gutenberg_experimental_global_styles_register_cpt() {
 }
 
 add_action( 'init', 'gutenberg_experimental_global_styles_register_cpt' );
-add_filter( 'block_editor_settings', 'gutenberg_experimental_global_styles_settings' );
 add_action( 'wp_enqueue_scripts', 'gutenberg_experimental_global_styles_enqueue_assets' );
+// This is to hook into the edit-post (block-editor store)
```

Comment:
Is this comment valid, in the sense that Is this hook specific only for `edit-post`? I would assume it would be triggered on every page that embeds the block editor.

I know that in WordPress core this hook is executed here:
https://github.com/WordPress/wordpress-develop/blob/165adab5e74cf3d593e1ac7cd5afa90a223c9a34/src/wp-admin/edit-form-blocks.php#L415

and that might be only related to the `edit-post` page. However, in practice, the name is so general that maybe we should ensure that those filters are registered based on the context.

Reply:
Ah, that comment is misleading, I've just removed it. That kind of control (which settings are passed depending on context) is already done within the function.